File: python/with_java/programmers/level 1/최대공약수와 최소공배수.py
def solution(n, m):
    최대공약수 = 1
    for num in range(2, min(n, m)+1):
        if n%num==0 and m%num==0:
            최대공약수 = num
    최소공배수 = 최대공약수 * (n/최대공약수) * (m/최대공약수)
    answer = [최대공약수, 최소공배수]
    return answer

File: java/src/main/java/ps_traning/programmers/level_1/최대공약수와_최소공배수.java
package ps_traning.programmers.level_1;

public class 최대공약수와_최소공배수 {
    public int[] solution(int n, int m) {
        int gcd = 1;
        for (int num = 2; num <= Math.min(n, m); num++) {
            if (n % num == 0 && m % num == 0) {
                gcd = num;
            }
        }

        int lcm = gcd * (n / gcd) * (m / gcd);
        int[] answer = {gcd, lcm};
        return answer;
    }
}
